### PR TITLE
Update vite

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: vite.config.mjs [hunk 1]
```diff
@@ -97,25 +99,29 @@ const pathExists = async (path) => {
 
 // https://vitejs.dev/config/
 export default defineConfig(async ({ mode }) => {
-  const scalaClassesDir = path.resolve(__dirname, `explore/target/scala-3.3.0`);
+
+  const _dirname = typeof __dirname !== 'undefined'
+    ? __dirname
+    : dirname(fileURLToPath(import.meta.url))
```
Comment (rpiaggio) [inline vite.config.mjs:105]: Why is this necessary?

**Review by rpiaggio — APPROVED** [review]
LGTM

File: vite.config.mjs (hunk 1 above)
Comment (cquiroz) [reply vite.config.mjs:105]: It is a change on vite 4.4.0 which requires to pass not a function but a value on the externaal build method.
this breaks where `_dirname` is available

See:
https://github.com/vitejs/vite/commit/998512b
https://github.com/vitejs/vite/issues/6946
